architecture/core_concepts/projects_and_users.adoc

```diff
@@ -134,3 +134,15 @@ endif::[]
 Developers and administrators can xref:../../dev_guide/projects.adoc#dev-guide-projects[interact
 with projects] using xref:../../cli_reference/index.adoc#cli-reference-index[the CLI] or the
 xref:../infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
+
+ifdef::openshift-online[]
+[[projects-hibenation]]
+=== Project Hibernation
+In {product-title} Starter, projects with 30 minutes of inactivity are placed in
+an idled state with resources scaled to zero. Upon receiving network activity,
+project resources are un-idled (scaled back up). In addition to auto-idling,
+projects must hibernate 18 hours in a 72-hour period. During the hibernation,
+all project resources are given a hard quota of zero (they cannot be scaled up).
```

Comment:
Numerals for the two "zeros" since there's > 10 numerals in the paragraph? [Stylepedia](http://stylepedia.net/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-N)